MetaCell/salk-interactive-atlas
# Feature/291-fix: Change the dot size of populations - NEW (after changes)

Closes #291 

Implemented solution: Tasks description
- Developed FE (no BE needed - since the dot size is managed in FE)
- change Dot size in threeDViewer with the help of DialogBox 
    - Change the slider value of dot size of each or all population(s).


How to test this PR: 
Steps:
- Toggle on any one or all population(s) from the Experiment sidebar.
- Dialog box appears.
- Slide through with mouse and change dotsize value. 
- Check if this is reflected in the threeDViewer.


Following are some of UI dev images:

![image](https://github.com/MetaCell/salk-interactive-atlas/assets/59233227/809c7ae3-45b9-4b7d-9806-7d3ae5f8928c)

![image](https://github.com/MetaCell/salk-interactive-atlas/assets/59233227/9540341d-aa36-4377-a001-014dee9dce10)



Video description:

Uploading 2023-11-06 07-51-03.mp4…




### Sanity checks:
- [] All the automated test checks are passing

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):

### Nice to have (if relevant):
